src/Shell/PluginShell.php

```diff
@@ -38,14 +40,20 @@ class PluginShell extends Shell
     public function getOptionParser()
     {
         $parser = parent::getOptionParser();
-
-        $parser->description(
-            'Plugin Shell perform various tasks related to plugin.'
-        )->addSubcommand('assets', [
-            'help' => 'Symlink / copy plugin assets to app\'s webroot',
-            'parser' => $this->Assets->getOptionParser()
-        ]);
+        $parser->addSubcommand(
```

Comment:
You need to add your parser options not overwrite existing ones. Restore back the description and command for assets task.